src/Controls/src/Core/Application/Application.cs

```diff
@@ -15,6 +16,7 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/Application.xml" path="Type[@FullName='Microsoft.Maui.Controls.Application']/Docs/*" />
+	[DebuggerDisplay("Windows Count: {Windows.Count}, MainPage = {MainPage}")]
```

Comment:
nit: why doesn't this follow the `GetDebuggerDisplay()` pattern?

Reply:
That wasn't supposed to be here, it was from my initial tests, will revert it